### ISSUE TITLE
Button cards not recognized as button by screen reader and not usable with keyboard

### ISSUE THREAD

**Issue by Nardol** [home-assistant/frontend#11611]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Using screen reader, button cards are not announced as a button. Moreover, it is not possible to activate them using keyboard only and access to them using tab.
I did not find any better way to use input buttons, using an entities card we only have a generic name.

### Describe the behavior you expected

When pressing Tab, they should be announced and as a button and we should be able to interact using keyboard.

### Steps to reproduce the issue

1. Add a button card, for example to press an input button
2. Navigate the interface and try to interact using keyboard

### What version of Home Assistant Core has the issue?

2022.2

### In which browser are you experiencing the issue with?

Every browsers

### Which operating system are you using to run this browser?

Debian Buster

### Problem-relevant frontend configuration

```yaml
type: button
  entity: input_button.chauffage_nuit
```

**Comment by Nardol** [comment]
Oops, edited for bad version number.
